### PR TITLE
Added link to blog post in readme.

### PR DESCRIPTION
**Pull request by KatiRG**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Front-end for [Participate](https://github.com/oliverbarnes/participate), a demo
 
 Stack: [Ember CLI](http://iamstef.net/ember-cli/) app, using the [Emblem](http://emblemjs.com) template engine, [EasyForm](https://github.com/dockyard/ember-easyForm), and [Mocha](http://visionmedia.github.io/mocha/) for testing.
 
-See the guide for [CONTRIBUTING](CONTRIBUTING.md), and the [CHANGELOG](CHANGELOG.md).
+See the guide for [CONTRIBUTING](CONTRIBUTING.md), and the [CHANGELOG](CHANGELOG.md). For more details on installation and setup, please see [this blog post](http://olibarnesdevstuff.tumblr.com/post/93398799975/getting-setup-with-ember-cli-emblem-easyform-mocha).
```
Comment (oliverbarnes) [inline README.md]: Stack installation makes more sense above, next to the Stack info. People might want to install without contributing.

Comment (KatiRG) [inline README.md]: OK i have added a Stack installation tip and updated the PR.

**Comment by oliverbarnes** [comment]
...squash commits ;)

**Comment by KatiRG** [comment]
done :)

**Pull request merged**